docs/sources/getting-started/getting-started-prometheus.md

```diff
@@ -0,0 +1,69 @@
++++
+title = "Getting started with Grafana and Prometheus"
+description = "Guide for getting started with Grafana"
+keywords = ["grafana", "intro", "guide", "started"]
+type = "docs"
+aliases = ["/docs/grafana/latest/guides/gettingstarted","/docs/grafana/latest/guides/getting_started"]
+[menu.docs]
+identifier = "getting_started-grafana-prometheus"
+parent = "guides"
+weight = 300
++++
+
+# Getting Started with Grafana and Prometheus
+
+Prometheus is an open-source systems monitoring and alerting toolkit for which Grafana includes built-in support. This topic walks you through the steps to create a series of dashboards in Grafana displaying system metrics of a host monitored by Prometheus.
+
+You can also configure your Grafana instance in the cloud to display system metrics. For more information, refer to the [Grafana Cloud](https://grafana.com/docs/grafana-cloud/) documentation.
```

Comment:
```suggestion
You can also configure your Grafana instance in the cloud to display system metrics. For more information, refer to [Grafana Cloud](https://grafana.com/docs/grafana-cloud/).
```

"documentation" feels redundant?

Reply:
With the proposed changes I'd expect to be taken to either the product itself or a product page, not documentation, so I'm not sure I agree about the redundancy.

Reply:
Then I'd suggest that we link to the relevant documentation section for configuring Grafana Cloud rather than the start page. For example:

> You can also configure your Grafana instance in the cloud to display system metrics. For more information, refer to [Metrics — Prometheus](https://grafana.com/docs/grafana-cloud/metrics/prometheus/).

"Metrics — Prometheus" isn't a great link text, but since we try to use the heading of the document we link to as link text, maybe we should consider renaming the heading to something more descriptive.

Reply:
@matthewhelmke, what are your thoughts on the use of "cloud" in this context?

Reply:
I would prefer, with a link to the cloud docs from "Grafana Cloud":

You have the option to configure a Grafana Cloud instance to display system metrics without having to host Grafana yourself.

Reply:
The direct answer to the question is "I'm not a fan." Hence my alternate suggestion. :smiley:

Reply:
Updated per Matthew's suggestion as we are talking about the clouds docs here.